fivetran/connectors/lambda/fivetran.tf

```diff
@@ -0,0 +1,18 @@
+resource "fivetran_connector" "lambda" {
```

Comment:
Will this resource be created regardless of having `sources_lambda` defined?

Reply:
If sources_lambda is not defined, the default value will be `[]` and nothing will be created